src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioActiveDocumentTracker.cs

```diff
@@ -228,7 +228,7 @@ public FrameListener(VisualStudioActiveDocumentTracker service, IVsWindowFrame f
                     {
                         _textBuffer = _documentTracker._editorAdaptersFactoryService.GetDocumentBuffer(bufferAdapter);
 
-                        if (!_textBuffer.ContentType.IsOfType(ContentTypeNames.RoslynContentType))
+                        if (_textBuffer != null && !_textBuffer.ContentType.IsOfType(ContentTypeNames.RoslynContentType))
```

Comment:
What was the bufferAdapter? It's not a regular editor buffer adapter but some sort of WinForms specific doc data?

Reply:
in my testing it was a `Microsoft.VisualStudio.Editor.Implementation.VsTextBufferAdapter` that we got from the forms designer.

![image](https://user-images.githubusercontent.com/9797472/88975192-a41fbb80-d26e-11ea-9dcb-207403d6ac23.png)

Reply:
So I don't think we can filter out getting a null buffer from the adapter type

Reply:
Not sure if it helps, but I used a VSShellUtilities method for getting a text buffer from a frame in CPS: https://devdiv.visualstudio.com/DevDiv/_git/CPS?path=%2Fsrc%2FMicrosoft.VisualStudio.ProjectSystem.VS.Implementation%2FEditor%2FProjectFrameOpenCloseListener.cs&version=GBmaster&line=223&lineEnd=228&lineStartColumn=1&lineEndColumn=1&lineStyle=plain